zbus/src/proxy/mod.rs

```diff
@@ -74,14 +75,22 @@ assert_impl_all!(Proxy<'_>: Send, Sync, Unpin);
 
 /// This is required to avoid having the Drop impl extend the lifetime 'a, which breaks zbus_xmlgen
 /// (and possibly other crates).
-#[derive(derivative::Derivative)]
-#[derivative(Debug)]
 pub(crate) struct ProxyInnerStatic {
-    #[derivative(Debug = "ignore")]
```

Comment:
This was done by @elmarco in 8a55ca77eb14fb9199f241b04f6bede9cf9fa567 and I see why but I wonder if it would be better to just make `Connection`'s debug repr better instead? :thinking: @elmarco ?

Reply:
let's assume that with silences, @elmarco signals his agreement. :)

Reply:
I still prefer that Proxy Debug doesn't provide Connection Debug.

Reply:
> I still prefer that Proxy Debug doesn't provide Connection Debug.

Can you elaborate a bit? More specifically, would you still prefer this even if connection debug was super slim? 

Also do you happen to remember, where the debug output was being verbose? If it's the tracing logs, I think we need to tell the intrument macro to skip `self` in places where we aren't already.

Reply:
@MaxVerevkin If you agree with me, please make the changes I suggest. Otherwise, let me know.

Reply:
I don't have an opinion on this one. Can we move it out of scope of this PR?

Reply:
Sure. My main concert is only that there is a lot of LOC in `proxy/mod.rs` already and I don't want to add more to that but perhaps we should split it anyways.